src/server/saved_objects/client/lib/create_find_query.js

```diff
@@ -32,19 +45,26 @@ export function createFindQuery(mappings, options = {}) {
   }
 
   if (search) {
+    // If searchFields is a singular string field, turn it into an array of one.
+    let searchFieldsArray = !searchFields || Array.isArray(searchFields) ? searchFields : [searchFields];
```

Comment:
Not a fan of hungarian notation, but can't think of a better way to approach this.